content/css/jenkins.css

```diff
@@ -853,27 +854,19 @@ ion-icon {
 }
 
 .jumbotron.plugins {
-  /* Permalink - use to edit and share this gradient: http://colorzilla.com/gradient-editor/#81b0c4+0,335061+100 */
-  background: #81b0c4; /* Old browsers */
-  background: -moz-linear-gradient(
-    left,
-    #81b0c4 0%,
-    #335061 100%
-  ); /* FF3.6-15 */
-  background: -webkit-linear-gradient(
-    left,
-    #81b0c4 0%,
-    #335061 100%
-  ); /* Chrome10-25,Safari5.1-6 */
-  background: linear-gradient(
-    to right,
-    #81b0c4 0%,
-    #335061 100%
-  ); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
-  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#81b0c4', endColorstr='#335061',GradientType=1 ); /* IE6-9 */
```

Comment:
We still want to offer support to old browsers for the time being, why are you deleting these?

Reply:
Removing these does not affect any browser released in the last decade, so this does not violate https://www.jenkins.io/doc/book/platform-information/support-policy-web-browsers/